app/javascript/member-facing/backbone/petition.js

```diff
@@ -17,6 +17,13 @@ const Petition = Backbone.View.extend({
     this.followUpUrl = options.followUpUrl;
     this.submissionCallback = options.submissionCallback;
     this.skipOnSuccessAction = options.skipOnSuccessAction;
+
+    if (options.redirectAfterAction !== undefined) {
+      this.redirectAfterAction = options.redirectAfterAction;
+    } else {
+      this.redirectAfterAction = true;
+    }
+
```

Comment:
LOL! I needed to add an option to disable the redirect after signing a petition. That's why I added the `redirectAfterAction` option; what I'm doing here is defaulting it to true. I know it looks a little to verbose, but I couldn't think of a shorter way to do this in javascript.